docs/required-fields.md

```diff
@@ -156,3 +160,4 @@ spec:
  ...
  ...
 ```
+
```

Comment:
nit: No need for this additional newline

Reply:
for once